wemake_python_styleguide/visitors/ast/keywords.py

```diff
@@ -280,12 +240,13 @@ def _check_variables_for_return(self, node: AnyFunctionDef) -> None:
                 ast.walk(node),
             ),
         )
-        assign = self._get_assign_node_variables(nodes)
-        names = self._get_name_nodes_variable(nodes)
-        returns, return_sub_nodes = self._get_return_node_variables(nodes)
+        assign = get_assign_node_variables(nodes)
+        names = get_name_nodes_variable(nodes)
+        returns, return_sub_nodes = get_return_node_variables(nodes)
```

Comment:
You can insert `breakpoint()` (if you are using 3.7, otherwise see https://realpython.com/python-debugging-pdb/) and debug this function.

Reply:
You can also use `ipbd`:
1. `pip install ipdb`
2. `export PYTHONBREAKPOINT=ipdb.set_trace`
3. Set `breakpoint()` where you need it
4. Run `flake8` on file that raises an unwanted violation